requirements.txt

```diff
@@ -0,0 +1,5 @@
+django-filter~=21.1.0
```

Comment:
Great that you pinned versions here!

Please add documentation to the readme how to create a virtual environment to use this configuration.